src/core/public/workspace/workspaces_service.mock.ts

```diff
@@ -14,11 +14,14 @@ const createWorkspacesSetupContractMock = () => {
   const workspaceList$ = new BehaviorSubject<WorkspaceObject[]>([]);
   const currentWorkspace$ = new BehaviorSubject<WorkspaceObject | null>(null);
   const initialized$ = new BehaviorSubject<boolean>(false);
+  const workspaceError$ = new BehaviorSubject<boolean>(false);
```

Comment:
```suggestion
  const workspaceError$ = new BehaviorSubject<string>('');
```
Should we make it an observable of string or error?

Reply:
Done